PIL/Image.py

```diff
@@ -1236,6 +1307,7 @@ def getpalette(self):
             return None  # no palette
 
     def getpixel(self, xy):
+        # type: (Coord) -> Tuple[Any]
```

Comment:
See the previous added code at the top of `Image.py`, now in `aliases.py`.

We could add `Color = float`, for example.